config/prow/config.yaml

```diff
@@ -597,13 +597,12 @@ tide:
             skip-unknown-contexts: true
   batch_size_limit:
     "kubernetes/kubernetes": 15
-#  TODO uncomment after merging #20468
-#  priority:
-#  # prioritize PRs fixing flaking/failing tests
-#  - labels: [ "kind/failing-test" ]
-#  - labels: [ "area/deflake" ]
-#  # prioritize critical bugfixes
-#  - labels: [ "kind/bug", "priority/critical-urgent" ]
+  priority:
+  # prioritize PRs fixing flaking/failing tests
+  - labels: [ "kind/failing-test" ]
+  - labels: [ "area/deflake", "priority/critical-urgent" ]
```

Comment:
Sorry, I should have been clearer :)

I think we should have this priority but will let @liggitt confirm since he opened https://github.com/kubernetes/test-infra/issues/18985

```yaml
  - labels: [ "kind/flake", "area/deflake", "priority/critical-urgent" ]
  - labels: [ "kind/failing-test", "priority/critical-urgent" ]
  - labels: [ "kind/bug", "priority/critical-urgent" ]
```

Reply:
ah ok go you now :)
yeah, if Jordan could step in...

Reply:
/assign @liggitt

Reply:
I have one concern here, but I'm not sure if it should block.

My read is that this is the same priority _for all tide queries_.  Is there any reason this wasn't implemented as a per-query setting?

I'm not sure that what `kubernetes/kubernetes` wants in terms of priority should dictate what `kubernetes/website` gets, for example.

Reply:
I don't have a strong opinion on requiring the priority labels either way, but I don't object. requiring both `"kind/flake"` and `"area/deflake"` is a bit tedious (it's not really clear to someone opening a PR to fix a bad flake they should add that area)

Reply:
I'm happy with any of the above iterations

/unassign

Reply:
> I'm not sure that what `kubernetes/kubernetes` wants in terms of priority should dictate what `kubernetes/website` gets, for example.

Good question... I don't think it's that critical because it's mostly k/k that has a huge tide backlog?

Reply:
>requiring both "kind/flake" and "area/deflake" is a bit tedious

Agreed. Drop the area label. Keep it to kind and priority. Those are labels consistent across the org

Otherwise yeah I think this is worth trying out

Reply:
@spiffxp done